sumologic-app-utils/src/sumoresource.py

```diff
@@ -1249,6 +1267,85 @@ def extract_params(self, event):
         return props
 
 
+class AlertsMonitor(SumoResource):
+
+    def _replace_variables(self, appjson_filepath, variables):
+        with open(appjson_filepath, 'r') as old_file:
+            text = old_file.read()
+            if variables:
+                for k, v in variables.items():
+                    text = text.replace("${%s}" % k, v)
+            appjson = json.loads(text)
+
+        return appjson
+
+    def _get_content_from_s3(self, s3url, variables):
+        with requests.get(s3url, stream=True) as r:
+            r.raise_for_status()
+            with tempfile.NamedTemporaryFile() as fp:
+                for chunk in r.iter_content(chunk_size=8192):
+                    if chunk:
+                        fp.write(chunk)
+                fp.flush()
+                fp.seek(0)
+                appjson = self._replace_variables(fp.name, variables)
+        return appjson
+
+    def _get_root_folder_id(self):
+        response = self.sumologic_cli.get_root_folder()
+        return response["id"]
+
+    def import_monitor(self, folder_name, monitors3url, variables, suffix_date_time):
+        date_format = "%d-%b-%Y %H:%M:%S"
+        root_folder_id = self._get_root_folder_id()
+        content = self._get_content_from_s3(monitors3url, variables)
+        content["name"] = folder_name + " " + datetime.utcnow().strftime(date_format) if suffix_date_time \
```

Comment:
is it expected? concatenating folder name with folder name if no suffix_date_time

Reply:
total condition is in else

Reply:
got it, I guess "if" has higher precedence over +.